Submissions/Byenna.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Juraia Pinas",
+    "img": "https://avatars.githubusercontent.com/u/72694655?v=4",
+    "email": "byenna21@gmail.com",
+    "links": {
+      "website": "juraia.nl",
+      "linkedin": "https://www.linkedin.com/in/juraia-pinas-b103a78b/",
+      "github": "https://github.com/Byenna"
+    },
+    "jobTitle": "[YOUR_TITLE]",
```

Comment:
Please remove placeholders, see guide in README

Reply:
Sorry, thought I covered everything. I corrected it.